src/jit/gentree.cpp

```diff
@@ -7122,7 +7106,8 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     printf("%c", (flags & GTF_CALL          ) ? 'C' : '-');
     printf("%c", (flags & GTF_EXCEPT        ) ? 'X' : '-');
     printf("%c", (flags & GTF_GLOB_REF      ) ? 'G' : '-');
-    printf("%c", (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');
+    printf("%c", (flags & GTF_MORPHED       ) ? '+' :
+                 (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');
```

Comment:
As written
First it see if it needs to print '+' for GTF_MORPHED
then it t doesn't IT prints 'O' for ORDER_SIDEEFF or '-' for no flag.

I also plan to propose shortly that we just deprecate the GTF_MORPHED flag entirely
and allow the global morph phase to morph nodes more than once.

Adding this now lets me see whether we are successful or not in have nodes morphed once.